packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -312,7 +312,7 @@ class SystemRulesTable extends React.Component {
     filterByPolicy = (policy, rows) => {
         const filteredRows = [];
         rows.forEach((row, i) => {
-            if (row.hasOwnProperty('isOpen') && policy.includes(row.cells[POLICY_COLUMN])) {
+            if (row.hasOwnProperty('isOpen') && policy.includes(row.cells[POLICY_COLUMN].toLowerCase())) {
```

Comment:
There was a bug here - when a report contained capitalized `severity` fields, they were not filtered properly with this. The filter was checking exclusive for lowercase: https://github.com/RedHatInsights/frontend-components/blob/master/packages/inventory-compliance/src/RulesComplianceFilter.js#L21 - so we can convert the severity here to lowercase to fix the bug.